app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -837,9 +840,8 @@ private View buildCatLabel(final String catName, ViewGroup categoryContainer) {
         if(!getString(R.string.detail_panel_cats_none).equals(catName)) {
             textView.setOnClickListener(view -> {
                 // Open Category Details page
-                String selectedCategoryTitle = CATEGORY_PREFIX + catName;
```

Comment:
The `CategoryClientKt.CATEGORY_PREFIX` static import in this file is not necessary anymore. So, it can be removed.